src/sig/picnic/sig_picnic.h

```diff
@@ -0,0 +1,90 @@
+#ifndef __OQS_SIG_PICNIC_H
+#define __OQS_SIG_PICNIC_H
+
+#include <oqs/oqs.h>
+
+#ifdef OQS_ENABLE_SIG_picnic_L1_FS
+
+#define OQS_SIG_picnic_L1_FS_length_public_key 33
+#define OQS_SIG_picnic_L1_FS_length_secret_key 49
+#define OQS_SIG_picnic_L1_FS_length_signature 34004
+
+OQS_SIG *OQS_SIG_picnic_L1_FS_new();
+
+extern OQS_STATUS OQS_SIG_picnic_L1_FS_keypair(uint8_t *public_key, uint8_t *secret_key);

```

Comment:
I don't think we need the `extern` keyword here or anywhere else in this file.

Reply:
This came from the nist-branch; should we remove it on both branches?

Reply:
There was a recent PR on nist-branch #387 that removed a bunch of superfluous extern's, but perhaps they missed some.  Please remove it here, and I'll ask Ben to do the same on nist-branch.

Reply:
I think a few algorithms didn't get cleaned of externs as there was active work on them. Chances are I forgot to add it in the PR while avoiding potential merge conflicts/jumbled history.